index.html

```diff
@@ -66,13 +67,11 @@ <h1>
 		<script async type="text/javascript" src="//cdn.carbonads.com/carbon.js?zoneid=1696&serve=CVYD42T&placement=dpilv" id="_carbonads_js"></script>
 		<!-- Carbon Zone Code -->
 	</aside>
-
-	<section id="devices">
+	<section id="devices" mv-app mv-storage="https://github.com/LeaVerou/dpi/blob/gh-pages/screens.json" class="mv-autoedit">
 		<h1>Known screens</h1>
-
-		<input type="search" placeholder="Search…" id="search" />
-
+		<input type="search" placeholder="Search…" id="search" property mv-storage="none"/>
 		<div id="screens">
+			<div class="mv-bar mv-ui" mv-bar="no-login"></div>
```

Comment:
You don't need an element for the Mavo bar if you don't plan to customize it. You can place the `mv-bar` attribute on the Mavo root.

Reply:
I thought it looked weird that the Mavo bar appeared on top of the "known screens" heading.